webroot/src/pages/SelectPrivileges/SelectPrivileges.ts

```diff
@@ -52,7 +50,26 @@ export default class SelectPrivileges extends mixins(MixinForm) {
     // Computed
     //
     get purchaseDataList(): Array<PrivilegePurchaseOption> {
-        return this.currentCompact?.privilegePurchaseOptions || [];
+        const privilegePurchaseOptions = [...this.currentCompact?.privilegePurchaseOptions || []];
+
+        privilegePurchaseOptions.sort((a, b) => {
+            let toReturn = 0;
+            const nameA = (a as PrivilegePurchaseOption).jurisdiction?.abbrev;
+            const nameB = (b as PrivilegePurchaseOption).jurisdiction?.abbrev;
```

Comment:
Would it make more sense to type the function arguments, rather than use the inline casting?

Also, it might make more sense to sort on what the user sees (state name) rather than the abbreviation. When doing that, it's usually best to also compare lower-cased versions of each string; that way if a state name is ever incorrectly updated to "arkansas" it won't get pushed to the top or bottom of the list.

Reply:
Oh whoops I should be sorting on the name and will do